crates/astria-sequencer/src/fees/state_ext.rs

```diff
@@ -475,6 +503,26 @@ pub(crate) trait StateWriteExt: StateWrite {
         self.put_raw(keys::IBC_SUDO_CHANGE.to_string(), bytes);
         Ok(())
     }
+
+    #[instrument(skip_all)]
+    fn delete_allowed_fee_asset<'a, TAsset>(&mut self, asset: &'a TAsset)
+    where
+        &'a TAsset: Into<Cow<'a, asset::IbcPrefixed>>,
+    {
+        self.nonverifiable_delete(keys::fee_asset(asset).into_bytes());
+    }
+
+    #[instrument(skip_all)]
+    fn put_allowed_fee_asset<'a, TAsset>(&mut self, asset: &'a TAsset) -> Result<()>
+    where
+        &'a TAsset: Into<Cow<'a, asset::IbcPrefixed>>,
+    {
+        let bytes = StoredValue::Unit
+            .serialize()
+            .context("failed to serialize unit for allowed fee asset")?;
+        self.nonverifiable_put_raw(keys::fee_asset(asset).into_bytes(), bytes);
```

Comment:
Discussed offline: this should be placed into the verifiable storage, not non-verifiable. (changing this will trigger changes to all gets/reads).